src/modules/game.js

```diff
@@ -45,11 +51,24 @@ class Game {
     }
 
     spawnEnemy() {
-        const enemy = new Enemy(0, 150, this, questionGenerator('insane'));
+        const enemy = new Enemy(
+            0,
+            this.randomLane(),
+            this,
+            questionGenerator('insane')
+        );
         this.gameBoard.element.appendChild(enemy.elements.enemy);
         this.enemies.push(enemy);
     }
 
+    // eslint-disable-next-line class-methods-use-this
+    randomLane() {
+        // randomly choose an object keys in the POSITION object
+        const keys = Object.keys(POSITION);
+        // eslint-disable-next-line no-bitwise
+        return POSITION[keys[(keys.length * Math.random()) << 0]];
```

Comment:
can we do this without using bitwise? confusing to read and linter doesn't like it.

Reply:
yap sure. I didn't understand that bitwise too but I saw that I think it's for rounding? and based on performance its better.